index.js

```diff
@@ -1,10 +1,11 @@
 var express = require("express")
+var config = require("./config.json")
 var app = express()
 
 app.get("/", function(req, res){
 	res.send("Test")
 })
 
-app.listen(3000, function(){
+app.listen(config.express.port, function(){
 	console.log("Listening on port 3000")
```

Comment:
Hmmm... I didn't update the port number in the string. Wonder how that happened :^)